mysql-test/r/transaction_isolation_change.result

```diff
@@ -0,0 +1,15 @@
+SET @saved_transaction_isolation = @@session.transaction_isolation;
+SET @saved_binlog_format = @@session.binlog_format;
+SET @@session.transaction_isolation = "READ-COMMITTED";
+SET @@session.binlog_format = ROW;
+CREATE TABLE t1 (a INT);
+CREATE TABLE t2 (a INT);
+REPLACE INTO t1(a) SELECT a FROM t2;
+SET @@session.transaction_isolation = "REPEATABLE-READ";
```

Comment:
Should we test other combinations of isolation levels too? Or is the disallowed combination only STATEMENT+READ_COMMITTED?

Reply:
Hi @satya-bodapati I don't think it depends on disallowed combinations logic. This fix causes flags recalculation regardless of what combination is allowed or not. 
In other words, we may theoretically say that combination A + B is not allowed from now on, and the fix will still work.

Reply:
@kamil-holubicki I understood the fix. Let me put my question in a different way.

Is there any other combination that fails after the regression commit and worked before the regression commit?

Reply:
Thank you kornelpal, This is exactly what I was looking for.
@kamil-holubicki can you please add this scenario to testcase.